feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -69,4 +76,13 @@ export class ConfiguratorAttributeProductCardComponent implements OnInit {
       valueCode: this.product.valueCode,
     });
   }
+
+  transformToProductType(value: Configurator.Value): Product {
+    return {
+      code: value.valueCode,
+      description: value.description,
+      images: {},
+      name: value.valueDisplay,
+    };
+  }
```

Comment:
@luklus : can you plz write a unit test for this new methods - will be a simple one.